pkg/kubernetes/views/replicationcontroller-panel.html

```diff
@@ -5,8 +5,8 @@
             <i class="fa pficon-replicator"></i>
         </div>
         <div class="listing-actions">
-            <button id="adjust-entity-btn" title="Adjust"  class="btn btn-default pficon pficon-edit"  data-toggle="modal" data-target="#adjust-entity-dialog"  data-id="{{id}}" data-key="{{item.key}}"></button>
-            <button id="delete-entity" title="Delete" class="btn btn-danger btn-delete pficon pficon-delete" data-toggle="modal" data-target="#delete-entity-dialog" data-id="{{id}}" data-key="{{item.key}}"></button>
+            <button id="adjust-rc-btn" title="Adjust"  class="btn btn-default pficon pficon-edit"  data-toggle="modal" data-target="#adjust-rc-dialog" data-key="{{item.key}}"></button>
```

Comment:
This should not have a an id='adjust-rc-btn'. This template is instantiated multiple times per page, and that would result in an invalid page.